homeassistant/components/teslemetry/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/teslemetry",
   "iot_class": "cloud_polling",
   "loggers": ["tesla-fleet-api"],
-  "requirements": ["tesla-fleet-api==0.9.2", "teslemetry-stream==0.5.3"]
+  "requirements": ["tesla-fleet-api==0.9.2", "teslemetry-stream==0.5.7"]
```

Comment:
Can we bump in a separate PR?

Reply:
Doing so now.

Reply:
https://github.com/home-assistant/core/pull/135344